bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/GC.java

```diff
@@ -195,7 +195,7 @@ void checkGC(int mask) {
 			if (data.gdipFgBrush != 0) Gdip.SolidBrush_delete(data.gdipFgBrush);
 			data.gdipFgBrush = 0;
 			long brush;
-			Pattern pattern = data.foregroundPattern;
+			Pattern pattern = data.foregroundPattern.getScaledPattern(data.shell);
```

Comment:
I think the PR is a bit old. Because it is already changed in the code. and I remember doing it before yesterday. Not sure why its showing up like that here.